app/components/nav-header.tsx

```diff
@@ -16,7 +16,7 @@ export function NavHeader() {
 
 	return (
 		<header className="p-2 border-b dark:border-neutral-700">
-			<div className="flex justify-between items-center">
+			<div className="flex flex-col xl:flex-row space-2 justify-between items-center">
```

Comment:
Default flex is row so no need for that. Set to `flex-col` for screens >xl. Also use gap-2 instead of space-2.

```suggestion
			<div className="flex max-xl:flex-col gap-2 justify-between items-center">
```